packages/async-flow/src/template.js

```diff
@@ -11,10 +11,21 @@ const BEST_GUESS_ID_REGEX = /^[a-zA-Z_$][a-zA-Z0-9_$]*$/;
  * Return an object that mimics a template strings array.
  *
  * @param {string[]} strings
+ * @param {string[]} [raw] optional raw strings, defaults to `strings`
  * @returns {TemplateStringsArray}
  */
-export const makeTemplateStringsArray = strings =>
-  harden(assign([...strings], { raw: strings }));
+export const makeTemplateStringsArray = (strings, raw = strings) => {
+  // Get the shape of the result object to match TemplateStringsArray.
+  const result = assign([...strings], { raw });
+
+  // Make the raw property non-enumerable.
+  defineProperty(result, 'raw', {
+    value: raw,
+    enumerable: false, // not needed, but good to be explicit
```

Comment:
x

Reply:
With the change, the explicit `enumerable: false,` is needed.
```suggestion
    enumerable: false,
```